samples/matter/lock/README.rst

```diff
@@ -183,10 +183,6 @@ The sample supports the following build types:
      - :file:`prj_thread_wifi_switched.conf`
      - nRF5340 DK with the nRF7002 EK shield attached
      - Debug version of the application with the ability to :ref:`switch between Thread and Wi-Fi network support <matter_lock_sample_wifi_thread_switching>` in the field.
-   * - No DFU
-     - :file:`prj_no_dfu.conf`
```

Comment:
I can't see this file removed in this sample.

Reply:
It's a kind of GitHub magic: 
![image](https://github.com/nrfconnect/sdk-nrf/assets/98917584/258d4bd5-bbb9-40f0-86e4-96df3bbea0c0)
:)

Reply:
wow :)